### ISSUE TITLE
0.14.0 problem description

### ISSUE THREAD

**Issue by donifer** [jdneo/vscode-leetcode#287]
## 🐛 Bug Report
Before 0.14.0 when I started a new problem the instructions would be in the new problem file commented out. But now I don't get the description in the problem's comments. I just get something like this without the description
```js
/*
 * @lc app=leetcode id=35 lang=javascript
 *
 * [35] Search Insert Position
 */
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function(nums, target) {
    
};


```

## To Reproduce
Start any problem from the sidenav.

## Expected behavior
The problems should include the description.

## Extension Output
```
/*
 * @lc app=leetcode id=35 lang=javascript
 *
 * [35] Search Insert Position
 */
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function(nums, target) {
    
};

[35] Search Insert Position  

https://leetcode.com/problems/search-insert-position/description/

Tags:   algorithms   array   binary-search 

Langs:  c   cpp   csharp   golang   java   javascript   kotlin   php   python   python3   ruby   rust   scala   swift 

* algorithms
* Easy (40.67%)
* Likes:    1225
* Dislikes: 170
* Total Accepted:    386.1K
* Total Submissions: 948.8K
* Testcase Example:  '[1,3,5,6]\n5'

<p>Given a sorted array and a target value, return the index if the target is found. If not, return the index where it would be if it were inserted in order.</p>

<p>You may assume no duplicates in the array.</p>

<p><strong>Example 1:</strong></p>

<pre>
<strong>Input:</strong> [1,3,5,6], 5
<strong>Output:</strong> 2
</pre>

<p><strong>Example 2:</strong></p>

<pre>
<strong>Input:</strong> [1,3,5,6], 2
<strong>Output:</strong> 1
</pre>

<p><strong>Example 3:</strong></p>

<pre>
<strong>Input:</strong> [1,3,5,6], 7
<strong>Output:</strong> 4
</pre>

<p><strong>Example 4:</strong></p>

<pre>
<strong>Input:</strong> [1,3,5,6], 0
<strong>Output:</strong> 0
</pre>

```

## Your Environment

- *os*: macOS
- *extension settings*: 
```json
{
  "leetcode.defaultLanguage": "javascript",
  "leetcode.showLocked": true,
  "leetcode.enableStatusBar": false,
  "leetcode.enableShortcuts": false
}
```
- *nodejs version*: 10.15.1
- *vscode version*: 1.33.1
- *extension version*: 0.14.0

**Comment by Vigilans** [comment]
We now rely on the preview panel to show problem description with better user experience:
![demo1](https://user-images.githubusercontent.com/20227484/56508589-4a065580-6557-11e9-91e6-b367363c3e52.gif)
![demo2](https://user-images.githubusercontent.com/20227484/56508592-4a9eec00-6557-11e9-8874-017ce5e0f077.gif)

Is there some reason for you to rely on the comment to provide description?

**Comment by donifer** [comment]
I normally have my problems in a repo and in my use case its nice to have the description in the file itself for future reference.

I can see myself going back to the problems a few years from now, maybe outside of vscode, and it would be nice to still have the description easily available.

Having an opt-in option to add the description to the file would be awesome!

**Comment by Vigilans** [comment]
I see. Will be back in next version.

**Comment by jdneo** [comment]
@Vigilans 

Since this is a breaking change we have introduced, how about we implement this and quickly release the `0.14.1` as a hotfix?

**Comment by Vigilans** [comment]
Yes, I'm currently working on it (to provide an opt-in option)

**Comment by jdneo** [comment]
@Vigilans Great! Thanks.

**Comment by juyse** [comment]
Also there should be easier way to access preview panel when re-open the code file, something similar to the "submit" button ?

**Comment by jdneo** [comment]
@juyse This is tracked by https://github.com/jdneo/vscode-leetcode/issues/286

**Comment by yanzhang0219** [comment]
When will you release 0.14.1 approximately? Is it possible to first release a version that at least supports this opt-in option to add descriptions in the problem's comment? Very desperate. Thanks a lot.

**Comment by jdneo** [comment]
Hi @all

## TL;DR
We have released `0.14.1` to address this issue. If you want to see the problem description in the code comments. Just open the VS Code `Preferences` search `leetcode.showCommentDescription` and turn it on.


## Flow Up
@Vigilans has also proposed a [PR](https://github.com/jdneo/vscode-leetcode/pull/293) to provide more information during using the extension, and I believe it should be included in the next release.

Sorry for introducing a breaking change this time. And thank you for your patience.

**Comment by miafan23** [comment]
@jdneo hello, the description window still open automatically after I have turn the `leetcode.showCommentDescription` option on, is there any way to disable it because the description have been displayed in the file already and I did not need it anymore.

**Comment by Vigilans** [comment]
@miafan23 Just out of doubt, what if there are pictures or highlighted characters in the description?

**Comment by Vigilans** [comment]
@miafan23 Maybe `showCommentDescription` should be changed to a setting with multiple choices: "No", "With webview", "Only in comment"

**Comment by jdneo** [comment]
@miafan23 Thank you for reporting.

Would you mind to create a new issue to track this?
